components/Storyblok/SbCtaLink.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { storyblokEditable, type SbBlokData } from '@storyblok/react';
+import { SbLink } from '@/components/Storyblok/partials/SbLink';
+import { type SbLinkType } from './Storyblok.types';
+import {
+  ctaLinkColors,
+  ctaLinkStyles,
+  ctaSizes,
+  type CtaLinkStylesType, 
+  type CtaLinkColorsType,
+  type CtaSizesType,
+  type LinkIconsType,
+} from '@/utilities/datasource';
+import { dcnb } from 'cnbuilder';
```

Comment:
On newer projects i just stick to cnb - didn't really like (and didn't feel the need to use) the dedupe so went with the faster cnb. If you need the dedupe here please feel free to keep it.